api/src/services/users.ts

```diff
@@ -164,7 +164,10 @@ export class UsersService extends ItemsService {
 	private inviteUrl(email: string, url: string | null): string {
 		const payload = { email, scope: 'invite' };
 
-		const token = jwt.sign(payload, getSecret(), { expiresIn: '7d', issuer: 'directus' });
+		const token = jwt.sign(payload, getSecret(), {
+			expiresIn: (env['USER_INVITE_TOKEN_TTL'] as string) ?? '7d',
```

Comment:
Default is already defined in `@directus/env` 👍 

```suggestion
			expiresIn: env['USER_INVITE_TOKEN_TTL'] as string,
```